_docs/_latest/setup/upgrade.md

```diff
@@ -1200,6 +1200,127 @@ This will recreate the VM with updated fields.
+            #
+            #Forseti Explain is not supported with a composite root at this time.
+            #
+            # Resources can exist in multiple organizations
+            #
+            #composite_root_resources:
+            #    - "folders/12345"
+            #    - "folders/45678"
+            #    - "projects/98765"
+            #    - "organizations/56789"
+
+       ```
+
+1. Rule files updates:
+  - Update [KMS rule file](https://github.com/GoogleCloudPlatform/forseti-security/blob/v2.13.0/rules/kms_rules.yaml)
+    under `rules/` in your Forseti server GCS bucket to be able to use the four
+    new use cases that have been added.
+  
+### Steps to upgrade using Terraform
+
+1. Update the version inside main.tf file to 1.3.0.
+1. Run command terraform init to get the plugins.
+1. Run command terraform plan to see the infrastructure plan.
+1. Run command terraform apply to apply the infrastructure build.
+
+{% endcapture %}
+{% include site/zippy/item.html title="Upgrading 2.12.0 to 2.13.0" content=upgrading_2_12_0_to_2_13_0 uid=13 %}
+
+{% capture deployment_manager_error %}
+
+If you get the following error while running the deployment manager:
+```
+The fingerprint of the deployment is .....
+Waiting for update [operation-xxx-xxx-xxx-xxx]...failed.
+ERROR: (gcloud.deployment-manager.deployments.update) Error in Operation [operation-xxx-xxx-xxx-xxx]: errors:
+- code: NO_METHOD_TO_UPDATE_FIELD
+  message: No method found to update field 'networkInterfaces' on resource 'forseti-server-vm-xxxxx'
+    of type 'compute.v1.instance'. The resource may need to be recreated with the
+    new field.
+```
+
+You can follow the following steps to workaround this deployment manager problem:
+1. Copy the compute engine section inside your deployment template and paste it to a text editor.
+    ```
+    # Compute Engine
+    - name: forseti-instance-server
+      type: forseti-instance-server.py
+      properties:
+        # GCE instance properties
+        image-project: ubuntu-os-cloud
+        image-family: ubuntu-1804-lts
+        instance-type: n1-standard-2
+        ...
+        run-frequency: ...
+    ```
+1. Remove the compute engine section from the deployment template.
+1. Run the deployment manager uppdate command on the updated deployment template.  
+`gcloud deployment-manager deployments update DEPLOYMENT_NAME --config forseti_server_v2_x_x.yaml`  
+This will remove your VM instance.
+1. Paste the compute engine section back to the deployment template, and run the update command again.  
+`gcloud deployment-manager deployments update DEPLOYMENT_NAME --config forseti_server_v2_x_x.yaml`  
+This will recreate the VM with updated fields.
+
+{% endcapture %}
+{% include site/zippy/item.html title="Error while running deployment manager" content=deployment_manager_error uid=50 %}
+
 {% endcapture %}
 {% include site/zippy/item.html title="Upgrading 2.X installations" content=2x_upgrade uid=1 %}
 
```

Comment:
@blueandgold, @red2k18  What should I put for whats next?

Reply:
You don't have to edit that section. It remains the same.